soil_id/services.py

```diff
@@ -100,7 +100,7 @@ def get_esd_data(ecositeID, esd_geo, ESDcompdata_pd):
 
 def get_soil_series_data(mucompdata_pd, OSD_compkind):
     series_name = [
-        re.sub("[0-9]+", "", compname)
+        re.sub("[0-9]+", "", compname).strip()
```

Comment:
question: how are valid soil series names formatted? what are the numbers we are stripping?